BWEnv/CMakeLists.txt

```diff
@@ -42,7 +42,11 @@ add_library(BWEnvObj OBJECT
   src/user_actions.cc
   src/utils.cc
   src/zmq_server.cc
-  ../replayer/frame.cpp
+  ../replayer/frame_serialization.cpp
+  ../replayer/keyframe.cpp
```

Comment:
Did we decide to leave this as frame.cpp?

Reply:
Yep. Reverted in P4. I'll merge P4 in directly -- P3 doesn't work on its own.